### DISCUSSION TITLE
Fixing old subs

### DISCUSSION THREAD

**Discussion by PiatraAlatyr** [FakeFishGames/Barotrauma#10890] [Ideas and suggestions]
Some older boats, such as the Kastrull, do not have crates attached to fabricators, although they are placed side by side. A small change that will please many players.

**Comment by MarkCringan** [comment]
Also, there are annoying "Dead Spaces" in the hulls of the Orca (ballasts) and Typhon (top airlock and junction room). Hulls inside the submarine, and even some stations, aren't connected properally, leading to random bits of ocean appearing it what should probably be a confined and dry area.
